roles/installer/tasks/23_rhcos_image_paths.yml

```diff
@@ -3,7 +3,7 @@
   block:
     - name: Get COMMIT_ID
       shell: |
-        /usr/local/bin/openshift-baremetal-install version | grep '^built from commit' | awk '{print $4}'
+        /usr/local/bin/{{ cmd }} version | grep '^built from commit' | awk '{print $4}'
```

Comment:
May we use TTP filter to get the commit instead of 'awk'. In case this command failed commit_id will get empty variable 
Also changed_when: false is recommended here
`

#cmd_openshift-baremetal-install.ttp
#{{ignore}} {{ version }}
#built from commit {{ commit }}
#release image {{ image }}
#release architecture {{ architecture }}

      changed_when: false
      command: |
        /usr/local/bin/{{ cmd }} version
      register: installer_built_from_commit

    - name: Parse openshift_baremetal_install command with TTP
      ansible.utils.cli_parse:
        text: "{{ installer_built_from_commit['stdout'] }}"
        parser:
          name: ansible.utils.ttp
          template_path: "{{ role_path }}/templates/cmd_openshift-baremetal-install.ttp"
        set_fact: openshift_baremetal_install_fact
        
 #commit_id_string: {{ openshift_baremetal_install_fact[0][0].commit }}
`

Reply:
Hi @reihl , thanks for the suggestion, since this is legacy code (pre 4.8) I rather not change it that much, besides this is long due to be removed and we expect it to be gone later this year.

Reply:
Maybe I create other PR to change it so we still get the advance from it. other then that please "set -o pipelinefail"